OpenEdXMobile/src/main/java/org/edx/mobile/view/LoginActivity.java

```diff
@@ -266,7 +266,8 @@ public void showErrorDialog(String header, String message) {
     }
 
     @Override
-    public boolean createOptionsMenu(Menu menu) {
+    protected void onResume() {
+        super.onResume();
```

Comment:
The issue seems to be that `BaseFragmentActivity` sets the default action bar properties in the `onStart()` callback, thus reverting any changes that the subclasses may have performed on each start. The proper solution would be to have the default properties set only once during the the initial `onCreate()` callback. Of course, this class should also do it from `onCreate()`, instead of the `createOptionsMenu()` callback, which isn't the right point to set up the action bar.